src/EditorFeatures/CSharpTest2/Recommendations/ForEachKeywordRecommenderTests.cs

```diff
@@ -13,190 +13,190 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Recommendations
     public class ForEachKeywordRecommenderTests : KeywordRecommenderTests
     {
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        public async Task TestAtRoot_Interactive()
+        public void TestAtRoot_Interactive()
         {
-            await VerifyKeywordAsync(SourceCodeKind.Script,
+            VerifyKeyword(SourceCodeKind.Script,
```

Comment:
❓ In the off chance we need to make this asynchronous in the future, can we keep the Verify methods asynchronous even though they currently run synchronously? It would save a lot of work both for this review and if something happens to change it again later.

Reply:
sure.

Reply:
Thanks this was a huge help!